src/Famix-Tests/FamixTWithInheritancesTest.class.st

```diff
@@ -14,3 +14,20 @@ self entity: (MooseEntity new copyWithTalent:FamixTWithInheritances).
 self entity class initializeSlots: self entity.
 model add: entity. 
 ]
+
+{ #category : #tests }
+FamixTWithInheritancesTest >> testSuperInheritances [
```

Comment:
The test comes from a trait I think and here you compiled it in the class using the trait I guess.